pnpm-lock.yaml

```diff
@@ -12168,6 +12286,49 @@ packages:
       - '@babel/core'
       - babel-plugin-macros
 
+  /next@13.4.12(react-dom@18.2.0)(react@18.2.0):
```

Comment:
And this is the other place. package.json has:
```
"next": "^13.0.6",
```

so it'll go up minor versions.